app/src/main/java/com/infomaniak/mail/ui/main/menuDrawer/MenuDrawerFragment.kt

```diff
@@ -357,6 +369,10 @@ class MenuDrawerFragment : Fragment() {
         mainViewModel.renameFolderResultTrigger.observe(viewLifecycleOwner) { modifyNameFolderDialog.resetLoadingAndDismiss() }
     }
 
+    private fun observeDeleteFolder() {
```

Comment:
We can merge these 3 functions in one, it's a bit overkill here to have them split up 😁